### DISCUSSION TITLE
Slideshow that includes sub-albums

### DISCUSSION THREAD

**Discussion by Puupuls** [LycheeOrg/Lychee#2828] [Ideas]
### Enhancement

Sometimes after a trip or event, its nice to organize pictures in sub-albums for nicer organisation. it would be nice if pressing slideshow button (or some menu choice) you could get it to play all images that are in this album tree

**Comment by ildyria** [comment]
It's an interesting idea. Unfortunately this would requiring adding a new endpoint and supporting pagination in slideshow.

I will leave this as project for volunteers because I really don't have the bandwidth to work on it.
